lib/taskana-core/src/main/java/pro/taskana/model/mappings/TaskMonitorMapper.java

```diff
@@ -52,14 +52,21 @@ long getTaskCountForWorkbasketByDaysInPastAndState(@Param("workbasketId") String
         @Param("status") List<TaskState> states);
 
     @Select("<script>"
-        + "SELECT WORKBASKET_KEY, COUNT(WORKBASKET_KEY) as counter "
+        + "<if test=\"_databaseId == 'db2'\">SELECT WORKBASKET_KEY, (DAYS(DUE) - DAYS(CURRENT_TIMESTAMP)) as AGE_IN_DAYS, COUNT(*) as NUMBER_OF_TASKS</if> "
+        + "<if test=\"_databaseId == 'h2'\">SELECT WORKBASKET_KEY, DATEDIFF('DAY', CURRENT_TIMESTAMP, DUE) as AGE_IN_DAYS, COUNT(*) as NUMBER_OF_TASKS</if> "

```

Comment:
What happens, if the current __databaseId_ is eighter "db2" or "h2"?
I´m guessing a _SqlException_ will be thrown - how do we want to deal with it?

_GROUPBY_ does work for each case, so it can be ignored as optional part.